views/chitter/home.erb

```diff
@@ -0,0 +1,27 @@
+<head>
+  <link href="https://fonts.googleapis.com/css?family=Oswald&display=swap" rel="stylesheet">
+  <style>
```

Comment:
It's great how you're thinking about how things will look like, as you're going along.